src/game/music.c

```diff
@@ -192,3 +194,19 @@ int16_t Music_CurrentTrack(void)
 {
     return m_Track;
 }
+
+int64_t Music_GetTimestamp(int16_t track)
+{
+    if (m_AudioStreamID < 0) {
+        return -1;
+    }
+    return S_Audio_StreamGetTimestamp(m_AudioStreamID);
+}
+
+bool Music_SeekTimestamp(int16_t track, int64_t timestamp)
```

Comment:
Maybe `int16_t track` should be removed if it's unused.

Reply:
I agree, but it seems now to be used

Reply:
Yes originally I didn't use it. The ffmpeg docs claimed using `-1` for `stream_index` uses the default stream, but it didn't actually ever seek. I only fixed the seeking by using the `track`.

https://ffmpeg.org/doxygen/trunk/group__lavf__decoding.html#gaa23f7619d8d4ea0857065d9979c75ac8